### PR TITLE
Add tests for testtools integration

### LINKED ISSUE #104: Makefile and tox.ini run different suites through twisted trial
Makefile tests trial against `tests/flexmock_pytest_test.py`:

https://github.com/flexmock/flexmock/blob/adc2419af992fe89aa21048aa8144f41baa16c99/Makefile#L47

tox tests trial against `tests/flexmock_test.py`:
https://github.com/flexmock/flexmock/blob/adc2419af992fe89aa21048aa8144f41baa16c99/tox.ini#L26

I'm assuming the Makefile should be changed to match (or deleted entirely).

### PR DESCRIPTION
**Pull request by ollipa**
I added simple tests for testtools integration.

Related issue #68 

Closes #104

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,21 +30,21 @@ pytest:
...
-	$(PYTHON) -m unittest tests/flexmock_test.py
+	$(PYTHON) -m unittest tests/test_flexmock.py
 
 .PHONY: twisted
 twisted:
 	@printf '\n\n*****************\n'
 	@printf '$(color)Running twisted tests$(off)\n'
 	@printf '*****************\n'
-	$(PYTHON) -c "from twisted.scripts.trial import run; run();" tests/flexmock_pytest_test.py
+	$(PYTHON) -c "from twisted.scripts.trial import run; run();" tests/test_flexmock.py
```
Comment (christophe-riolo) [inline Makefile:48]: Should a testtools section be added ?

**Review by christophe-riolo — COMMENTED** [review]
There is a weird warning 
```
RuntimeWarning: 'testtools.run' found in sys.modules after import of package 'testtools', but prior to execution of 'testtools.run'
```
but I think it's not kinked to the code, more in the structure of testtools modules, so :shrug:

**Review by christophe-riolo — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (ollipa) [reply Makefile:48]: I was thinking that maybe `make test` could just run unittest, doctest and pytest. Then a separate command `make test-integration` could run all the other integrations. CI pipeline will anyway run all the integration tests. What do you think?

Comment (christophe-riolo) [reply Makefile:48]: Yeah I am not sure that we need to put everything under the makefile. I am not sure we need the makefile altogether even :D Let's move the discussion to an issue :)

Comment (ollipa) [reply Makefile:48]: https://github.com/flexmock/flexmock/issues/116

**Pull request merged**